packages/jwt-verifier/test/spec/verifier.spec.js

```diff
@@ -86,11 +90,14 @@ describe('Jwt Verifier', () => {
       });
     }, LONG_TIMEOUT);
 
-    it('should fail if no kid is present in the JWT header', () => {
+    it('should fail if no kid is present in the JWT header and keys endpoint provides > 1 key', () => {
```

Comment:
to align with [jwks-rsa update](https://github.com/communitybuilders/node-jwks-rsa/blob/31c7f654766e96996fcefec08db33e65605b54bb/src/JwksClient.js#L112)